osm_jet/fields.py

```diff
@@ -32,3 +45,134 @@ class DjangoManyToMany(models.ManyToManyField):
     def formfield(self, **kwargs):
         kwargs['widget'] = DjangoManyToManyWidget
         return super().formfield(**kwargs)
+
+
+class SortedManyToManyWidget(CheckboxSelectMultiple):
+    '''
+        A customised version of the sortedM2M SortedCheckboxSelectMultiple.
+
+        Add the current model to the context so that we can have 'edit' links
+        in the select multiple.
+    '''
+    template_name = 'sortedm2m/sorted_checkbox_select_multiple_widget.html'
+    model_cls = None
```

Comment:
`model_cls` should be an instance attribute, not a class attribute; you can end up with a nasty race condition this way.